### ISSUE TITLE
[Bug] Flight Modes

### ISSUE THREAD

**Issue by Xerma** [tsunamayo/Starship-EVO#4516]
### Short Description of the issue:
I'm not sure how, maybe it has something to do with the recent multiplayer fixes, but while in photo mode the flight mode seems to only be 6dof (even when the mode is at anti-gravity while not in photo mode). Previously photo mode followed what the player flight mode was in.

Another thing is anti-gravity mode feels like it has a lot more rotation than it should have/what it used to have

### Environment
- Starship EVO version: 22w04b
- Graphics Card: GTX 970

### Additional information:
I know these screenshots say 22w04a but I did update to 22w04b and this still applies

![20220125214139_1](https://user-images.githubusercontent.com/32974671/151101217-cc155682-a917-4a92-ac67-020236b4ff68.jpg)
![20220125214145_1](https://user-images.githubusercontent.com/32974671/151101223-87f80634-9b51-49cd-9be1-a405a71dc2a0.jpg)
![20220125214712_1](https://user-images.githubusercontent.com/32974671/151101284-34b869d4-08ba-4562-8ada-583f597dd13d.jpg)
![20220125214714_1](https://user-images.githubusercontent.com/32974671/151101298-1f5cb1e4-db16-481c-ad4a-81440932d7e2.jpg)
![20220125214715_1](https://user-images.githubusercontent.com/32974671/151101305-9275a890-0929-4ee0-84b9-a0b62e281ddd.jpg)

**Comment by tsunamayo** [comment]
@Xerma yep, it was more logical to do it that way. I fixed a bunch of issue with photo mode also, 6dof was broken...
In photomode jetpack mode is not really making sense, if you want to move like with the player locomotion logic dont use normal mode I guess ^^ ps: this bad boi is coming along I see...

**Comment by Xerma** [comment]
@tsunamayo I just realized that for some reason during the last update, if you had anti-gravity set as your flight mode it would behave like 6dof until you cycle through the flight modes again.

This seems just like a 1 timer bug. I cycled through the modes, back to anti-gravity, and confirm it behaves as normal. Exiting and reloading the game I can also confirm that this issue doesn't persist.

Just some oddity it seems.

As for photo mode, having it be 6dof isn't the worst thing in the world but is there anyway we can have it remain in either anti-gravity or 6dof? If the player is in jetpack mode then it could just default to 6dof (Or just persist the flight mode toggle through to when the player is in photo mode) Although if the player is in jetpack mode then anti-gravity might make more sense as a default since the player must be in gravity so locking to that plane would make more sense, to me anyways.
